src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String fruitName;
```

Comment:
The class name does not need to be added to the fields
```suggestion
    private String name;
```